crates/sui-graphql-rpc/src/types/address.rs

```diff
@@ -17,6 +17,7 @@ use super::{
     transaction_block::{TransactionBlock, TransactionBlockFilter},
 };
 
+/// The 32-byte address that is an account address (corresponding to a public key).
```

Comment:
nit: I would personally remove this copy of the comment, to avoid things going out of sync. It's a little bit funky then that for GraphQL code the doc comment is in a strange place, but it avoid people accidentally updating just the type doc comment, or otherwise not updating both of these.

Reply:
That makes sense, they always get out of sync.